tui/network.py

```diff
@@ -286,23 +373,37 @@ def specify_configuration(answers, txt, defaults):
         ifaceName = conf_dict['config'].getInterfaceName(conf_dict['interface'])
         netutil.ifdown(ifaceName)
 
-        # check that we have *some* network:
-        if netutil.ifup(ifaceName) != 0 or not netutil.interfaceUp(ifaceName):
+        def display_error():
```

Comment:
also reindenting in the same commit - all such new-func reindentations could live in a single separate commit